app.js

```diff
@@ -7,8 +7,10 @@ const fs = require('fs');
 const app = require('./server');
 
 const isDevelopment = app.get('env') === 'development';
+const hasCertificate = fs.existsSync(path.resolve(__dirname, 'key.pem'))
+	&& fs.existsSync(path.resolve(__dirname, 'cert.pem'));
 
-if(isDevelopment) {
+if(isDevelopment && hasCertificate) {
```

Comment:
In Heroku review apps, we are not production, but also don't have self-signed certificates.